src/package/package.json

```
@@ -0,0 +1,20 @@
+{
+    "name": "trafficcop",
```

Question: should name be `@mozmeao/trafficcop`? since we're using an NPM namespace?
```suggestion
    "name": "@mozmeao/trafficcop",
```